### DISCUSSION TITLE
Como faço para encontrar link m3u para copiar e colar em aplicação "IPLAYTV" na Apple TV?

### DISCUSSION THREAD

**Discussion by marcosferreira14** [inspirationlinks/lista-tuga#5] [Q&A]
Saquei uma aplicação de IPTV "IPLAYTV" na apple tv e gostava de usar este m3u. Não percebo nada de iptv's mas já experimentei alguns links m3u e deram, mas este git não me mostra o link diretamente, o que tenho de fazer? Obrigado

**Comment by iprtl** [comment]
queres o link directo é isso?? tipo assim?? https://raw.githubusercontent.com/inspirationlinks/m3u/live/pt.m3u8

**Reply by joedualmar** [reply]
funciona no iboplayer pro ?

**Comment by iprtl** [comment]
> Saquei uma aplicação de IPTV "IPLAYTV" na apple tv e gostava de usar este m3u. Não percebo nada de iptv's mas já experimentei alguns links m3u e deram, mas este git não me mostra o link diretamente, o que tenho de fazer? Obrigado

Boas. Para ser sincero, só experimentando. Não conheço aplicação em questão.
